### ISSUE TITLE
Possible font spacing issue

### ISSUE THREAD

**Issue by vladvasiliu** [tomszilagyi/zutty#37]
Hi,

I've been looking to try Zutty and have noticed a possible font spacing issue with a "skinny" font, Iosevka. The glyphs seem to have the proper shape, but the spacing is off. If I use a "regular" font, for instance Terminus (bitmap) or Adobe Source Code Pro, everything looks OK. Is this something related to the font, or something else?

I've started Zutty with :

```
zutty -font iosevka-ss09-regular -fontsize 28
```

Zutty was installed from the Arch Linux community repository. Version is 0.9, built from [commit 9f30b58cff891475c3bcb56cb8c91db24b71a617](https://github.com/tomszilagyi/zutty#commit=9f30b58cff891475c3bcb56cb8c91db24b71a617).

Below is a comparison of Alacritty and Zutty, on the left and right, respectively.

![image](https://user-images.githubusercontent.com/175579/117543641-f839a680-b01d-11eb-867c-6e1f8ed1853c.png)

**Comment by tomszilagyi** [comment]
Hi,

Can you please try with the `-fixed` variant?
I manually installed from this archive (I do not use Arch) and it seems to work fine:
https://github.com/be5invis/Iosevka/releases/download/v3.0.0/ttf-iosevka-fixed-ss09-3.0.0.zip

Even better, if you leave off the `-regular` suffix, then Zutty will figure out where different variants are and use them:
```$ zutty -v -font iosevka-fixed-ss09 -fontpath ~/doc/fonts -fontsize 20
Zutty 0.9
Copyright (C) 2020 Tom Szilagyi

This program comes with ABSOLUTELY NO WARRANTY.
Zutty is free software, and you are welcome to redistribute it
under the terms and conditions of the GNU GPL v3 (or later).

I [font.cc: 55] Loading /home/tom/doc/fonts/ttf/iosevka-fixed-ss09-regular.ttf as primary
I [font.cc:214] Pixel size 20
I [font.cc:225] Glyph size 10x25
I [font.cc:133] Skipped loading 359 code point(s) outside the Basic Multilingual Plane
I [font.cc: 55] Loading /home/tom/doc/fonts/ttf/iosevka-fixed-ss09-bold.ttf as overlay
I [font.cc:214] Pixel size 20
I [font.cc:225] Glyph size 10x25
I [font.cc:133] Skipped loading 359 code point(s) outside the Basic Multilingual Plane
I [font.cc: 55] Loading /home/tom/doc/fonts/ttf/iosevka-fixed-ss09-italic.ttf as overlay
I [font.cc:214] Pixel size 20
I [font.cc:225] Glyph size 10x25
I [font.cc:133] Skipped loading 359 code point(s) outside the Basic Multilingual Plane
I [font.cc: 55] Loading /home/tom/doc/fonts/ttf/iosevka-fixed-ss09-bolditalic.ttf as overlay
I [font.cc:214] Pixel size 20
I [font.cc:225] Glyph size 10x25
I [font.cc:133] Skipped loading 359 code point(s) outside the Basic Multilingual Plane
```

Nice font, by the way!

**Comment by vladvasiliu** [comment]
Thanks, the fixed variant works.
